packages/fx-core/src/component/driver/aad/utility/aadManifestHelper.ts

```diff
@@ -386,4 +388,12 @@ export class AadManifestHelper {
     await updateVersionForTeamsAppYamlFile(projectPath);
     return ok(undefined);
   }
+
+  public static isTestToolEnabledProject(projectPath: string): boolean {
```

Comment:
Seems this is not an AAD manifest related function, do we have a better place for it?